daemon/cmd/status.go

```diff
@@ -200,6 +210,82 @@ func (d *Daemon) getKubeProxyReplacementStatus() *models.KubeProxyReplacement {
 	}
 }
 
+func (d *Daemon) getBPFMapStatus() *models.BPFMapStatus {
+	return &models.BPFMapStatus{
+		DynamicSizeRatio: option.Config.BPFMapsDynamicSizeRatio,
+		Maps: []*models.BPFMapProperties{
+			{
+				Name: "Non-TCP connection tracking",
+				Size: int64(option.Config.CTMapEntriesGlobalAny),
+			},
+			{
+				Name: "TCP connection tracking",
+				Size: int64(option.Config.CTMapEntriesGlobalTCP),
+			},
+			{
+				Name: "Endpoint policy",
+				Size: int64(lxcmap.MaxEntries),
+			},
+			{
+				Name: "Events",
+				Size: int64(eventsmap.MaxEntries),
+			},
+			{
+				Name: "IP cache",
+				Size: int64(ipcachemap.MaxEntries),
+			},
+			{
+				Name: "IP masquerading",
+				Size: int64(ipmasqmap.MaxEntries),
+			},
+			{
+				Name: "IPv4 fragmentation",
+				Size: int64(option.Config.FragmentsMapEntries),
+			},
+			{
+				Name: "Metrics",
+				Size: int64(metricsmap.MaxEntries),
+			},
+			{
+				Name: "NAT",
+				Size: int64(option.Config.NATMapEntriesGlobal),
+			},
+			{
+				Name: "Neighbor table",
+				Size: int64(option.Config.NeighMapEntriesGlobal),
+			},
+			{
+				Name: "Global policy map",
+				Size: int64(option.Config.PolicyMapEntries),
+			},
+			{
+				Name: "Per endpoint policy",
+				Size: int64(eppolicymap.MaxEntries),
+			},
+			{
+				Name: "Session affinity",
+				Size: int64(lbmap.MaxEntries),
+			},
+			{
+				Name: "Signal",
+				Size: int64(signalmap.MaxEntries),
+			},
+			{
+				Name: "Sockmap",
+				Size: int64(sockmap.MaxEntries),
+			},
+			{
+				Name: "Sock reverse NAT",
+				Size: int64(option.Config.SockRevNatEntries),
+			},
+			{
+				Name: "Tunnel",
+				Size: int64(tunnelmap.MaxEntries),
+			},
```

Comment:
Although lbmap sizes are static, but for completion I'd suggest to add the following maps: `Service{4,6}MapV2`, `Backend{4,6}Map`, `RevNat4Map`.

Reply:
Added, PTAL.